javalin/src/main/java/io/javalin/plugin/json/JsonMapper.kt

```diff
@@ -13,4 +13,3 @@ import java.io.InputStream
 const val JSON_MAPPER_KEY = "global-json-mapper"
 fun Javalin.jsonMapper(): JsonMapper = this.attribute(JSON_MAPPER_KEY)
 fun Context.jsonMapper(): JsonMapper = this.appAttribute(JSON_MAPPER_KEY)
-fun JsonMapper.canReadStream() = !this::class.java.getMethod("fromJsonStream", InputStream::class.java, Class::class.java).isDefault
```

Comment:
I'm quite glad this isn't here anymore tbh 😅

Reply:
I don't disagree 😅